cozumler/en-yakin-noktalar/java/EnYakinNoktalar.java

```diff
@@ -0,0 +1,28 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class EnYakinNoktalar {
+	public static void main(String[] args)
+	  {
```

Comment:
indentation ve space sorunlari var gibi duruyor. genel olarak.

Reply:
Bu aslında eclipse'in var olan düzenleme stili. Fakat tekrar düzenleyeceğim teşekkürler.